mylfs.sh

```diff
@@ -899,72 +1012,10 @@ then
     EXTENSION="$(cd $(dirname $EXTENSION) && pwd)/$(basename $EXTENSION)"
 fi
 
-
-# ###########
-# Start build
-# ~~~~~~~~~~~
-
-# Perform single operations
-$CHECKDEPS && check_dependencies && exit
-$DOWNLOAD && download_pkgs && exit
-$INIT && download_pkgs && init_image && unmount_image && exit
-$MOUNT && mount_image && exit
-$UNMOUNT && unmount_image && exit
-$CLEAN && clean_image && exit
-[ -n "$INSTALL_TGT" ] && install_image && exit
-
-if [ -n "$STARTPHASE" ]
-then
-    download_pkgs
-    mount_image
-elif $BUILDALL
+if $BUILDISO
 then
-    download_pkgs
-    init_image
+    startISO
 else
-    usage
-    exit
+    mainLFSbuild $STARTSECTION
 fi
 
-PATH=$LFS/tools/bin:$PATH
-CONFIG_SITE=$LFS/usr/share/config.site
-LC_ALL=POSIX
-export LC_ALL PATH CONFIG_SITE
-
-trap "echo 'build cancelled.' && cd $FULLPATH && unmount_image && exit" SIGINT
-trap "echo 'build failed.' && cd $FULLPATH && unmount_image && exit 1" ERR
-
-build_phase 1
-
-$ONEOFF && $FOUNDSTARTPHASE && unmount_image && exit
-
-build_phase 2
-
-$ONEOFF && $FOUNDSTARTPHASE && unmount_image && exit
-
-build_phase 3
-
-# phase 3 cleanup
-rm -rf $LFS/usr/share/{info,man,doc}/*
-find $LFS/usr/{lib,libexec} -name \*.la -delete
-rm -rf $LFS/tools
-
-$ONEOFF && $FOUNDSTARTPHASE && unmount_image && exit
-
-build_phase 4
-
-$ONEOFF && $FOUNDSTARTPHASE && unmount_image && exit
-
-[ -n "$EXTENSION" ] && build_extension
-
-# final cleanup
-rm -rf $LFS/tmp/*
-find $LFS/usr/lib $LFS/usr/libexec -name \*.la -delete
-find $LFS/usr -depth -name $LFS_TGT\* | xargs rm -rf
-rm -rf $LFS/home/tester
-sed -i 's/^.*tester.*$//' $LFS/etc/{passwd,group}
-
-# unmount and detatch image
-unmount_image
-
-echo "build successful."
```

Comment:
I like that you moved this into a separate function

Reply:
I moved it cause `iso` will work separately from building

Reply:
yeah makes complete sense, even without the iso build